com.unity.render-pipelines.high-definition/Runtime/Sky/AmbientProbeConvolution.compute

```diff
@@ -12,12 +12,12 @@ TEXTURECUBE(_AmbientProbeInputCubemap);
 
 // Constants from SetSHEMapConstants function in the Stupid Spherical Harmonics Tricks paper:
 // http://www.ppsloan.org/publications/StupidSH36.pdf
-#define sqrtPI (sqrt(PI))
-#define fC0 (0.282095 * (1.0 / (2.0 * sqrtPI)))
-#define fC1 (0.488603 * (sqrt ( 3.0) / ( 3.0 * sqrtPI)))
-#define fC2 (1.092548 * (sqrt (15.0) / ( 8.0 * sqrtPI)))
-#define fC3 (0.315392 * (sqrt ( 5.0) / (16.0 * sqrtPI)))
-#define fC4 (0.546274 * 0.5 * fC2)
```

Comment:
@JulienIgnace-Unity : this one was incorrect
#define fC4 (0.546274 * 0.5 * fC2)

it must be #define fC4 (0.546274 * 0.5 * (sqrt (15.0) / ( 8.0 * sqrtPI))) without the 1.092548